recipes/client.rb

```diff
@@ -8,6 +8,16 @@
 pkgs = node['percona']['client']['packages'].empty? ? percona_client_packages : node['percona']['client']['packages']
 pkgs << percona_devel_package if node['percona']['client']['install_devel_package']
 
+# If we are installing 8.0 on a Centos 7 instance we need to remove the existing
+# mariadb-libs rpm to avoid a dependency conflict.
+package 'mariadb-libs' do
+  action :remove
+  only_if { percona_8_on_centos_7 }
+end
+
 package pkgs do
   action node['percona']['client']['package_action'].to_sym
 end
+
+# Unfortunately, removing mariadb-libs also removes postfix...which we want.
+package 'postfix'
```

Comment:
This isn't always going to be the case. I think given this one-off a better method might be to add a `raise` if `mariadb-libs` exists on a CentOS 7 while trying to install Percona 8.0. We shouldn't be installing a specific MTA.

Not sure what @damacus thinks however, just doesn't feel right to me. For the purpose of CI we can just remove it in a test cookbook however.